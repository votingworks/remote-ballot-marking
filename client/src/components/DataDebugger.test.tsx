import React from 'react'
import { render } from '@testing-library/react'

import DataDebugger from './DataDebugger'

it('renders DataDebugger', () => {
  const { container } = render(
    <DataDebugger
      data={{
        foo: 'bar',
      }}
    />
  )
  expect(container.firstChild).toMatchSnapshot()
})

it('hides DataDebugger', () => {
  const { container } = render(
    <DataDebugger
      hide
      data={{
        foo: 'bar',
      }}
    />
  )
  expect(container.firstChild).toMatchSnapshot()
})
