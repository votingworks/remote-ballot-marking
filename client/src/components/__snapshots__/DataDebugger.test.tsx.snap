// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hides DataDebugger 1`] = `null`;

exports[`renders DataDebugger 1`] = `
<pre>
  {
  "foo": "bar"
}
</pre>
`;
